npm/webpack-dev-server/src/helpers/vueCliHandler.ts

```diff
@@ -17,8 +17,6 @@ export function vueCliHandler (devServerConfig: WebpackDevServerConfig): PresetH
 
     debug('webpack config %o', webpackConfig)
 
-    delete webpackConfig.entry
```

Comment:
Instead of deleting it in many places, and then having to remember this if we implement a new webpack handler, any chance we could lift this up higher and do something like

```js
if (!angular) {
  delete webpackConfig.entry
}
```

Thoughts?

Reply:
This PR is doing exactly that. In #22314 I moved entry handling to each handler. This PR reverts those changes so that the handlers don't have to worry about entrypoints since they will be replaced with the Cypress entry [here](https://github.com/cypress-io/cypress/pull/23299/files#diff-7031da07fd51256fb2dc14db39454b4e8ec0f8f90ab283511c2bf847e790eead)

The line you referenced is a removal, not an addition (unless I'm misunderstanding your comment)